packages/common/src/items/create-item-from-url-or-file.ts

```diff
@@ -74,5 +87,10 @@ export async function createItemFromUrlOrFile(
     );
   }
 
-  return createdItem;
+  return {
```

Comment:
I know it's not part of this PR, but do you have some `try/catch` stuff inside the `createItemFrom...` fns? I see a `failSafe(...)` on the share call, but it looks like the others could fail and that could cascade poorly

Reply:
Like I don't see any `Promise.reject(...)` in the `create-item-...` test below.

Reply:
Sub funcs do have try/catch.